content/en/docs/operations/oidc/enable_oidc.md

```diff
@@ -9,7 +9,7 @@ aliases:
 
 ## Prerequisites
 
-1. **OIDC Configuration**  
```

Comment:
The double trailing space was here for a reason; it's a line break. I'll put explicit `<br/>` there.

Reply:
Oh! Did not know that, sorry to mix changes together